render.js

```diff
@@ -0,0 +1,64 @@
+
+function render(dataOrString, renderer, configuration) {
+
+    /* Convert a string to the BCD data */
+    let data = undefined;
+    if (typeof dataOrString === 'string' || dataOrString instanceof String) {
+        const dataParts = dataOrString.split('.');
+        data = require('.');
+        dataParts.forEach((elem) => {
+            data = data[elem] || {};
```

Comment:
Instead of returning empty object in case the query finds no data, we should probably return `console.error("No data found.")` or similar.